site/content/blog/2024-10-24-zap-updates-october-2024/index.md

```diff
@@ -0,0 +1,181 @@
+---
+title: "ZAP Updates - October 2024"
+summary: >
+TODO
```

Comment:
Should this be indented? Is that why the build failed?

Reply:
Yeah, tweaked in https://github.com/zaproxy/zaproxy-website/pull/2843.